test/create-open.test.js

```diff
@@ -141,13 +141,6 @@ Object.keys(testAPIs).forEach(API => {
           assert.equal(manifest.accessController.indexOf('/ipfs'), 0)
         })
 
-        it('can pass local database directory as an option', async () => {
-          const dir = './orbitdb/tests/another-feed'
-          db = await orbitdb.create('third', 'feed', { directory: dir })
-          localDataPath = path.join(dir, db.address.root, db.address.path)
-          assert.equal(fs.existsSync(localDataPath), true)
-        })
-
```

Comment:
Why remove this test?

Reply:
Since the store is persistent, we no longer pass the directory in at `orbitdb.open` time, instead passing it at `OrbitDB.createInstance` time.

Reply:
Got it 👍